roles/keystone-setup/tasks/main.yml

```diff
@@ -50,6 +51,20 @@
                  token={{ admin_token }}
   with_items: keystone.user_roles
 
+- name: heat stack user
+  keystone_user: user=heat_stack_user
+                 password="{{ secrets.service_password }}"
+                 tenant=admin
+                 token={{ admin_token }}
+  when: heat.enabled|bool
+
+- name: heat stack user role
+  keystone_user: role=heat_stack_user
```

Comment:
We'll want to move over to the new upstream module for doing users in keystone, but that can be a later PR.